### ISSUE TITLE
Pagination in search results broken

### ISSUE THREAD

**Issue by sanderha** [silverstripe/silverstripe-fulltextsearch#228]
I might be missing something, but it seems generated search results does not care about the "start" GET variable.
Shouldnt this method `SilverStripe\FullTextSearch\Solr\Forms\SearchForm::getResults()` account for that?
I fixed it in my own `SearchForm` class by adding this:
```
        $searchTerms = $request->requestVar('Search');
        $start = $request->requestVar('start');
        $query = SearchQuery::create()->addSearchTerm($searchTerms)->setStart($start);
```

Apologies if I missed something obvious

Using SS v4.1 and v3.3 of this module

**Comment by robbieaverill** [comment]
@sanderha would you like to make a pull request to fix this?

**Comment by sanderha** [comment]
#229 There ya go

**Comment by robbieaverill** [comment]
Fixed in #229, thank you
